target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -81,6 +81,16 @@ dlink,dwr-960)
 	ucidef_set_led_switch "lan" "lan" "green:lan" "switch0" "0x2e"
 	ucidef_set_led_switch "wan" "wan" "green:wan" "switch0" "0x01"
 	;;
+domywifi,dm202|\
+domywifi,dm203|\
+domywifi,dw22d)
+	ucidef_set_led_switch "lan1" "lan1" "amber:lan1" "switch0" "0x02" "" "link"
+	ucidef_set_led_switch "lan2" "lan2" "amber:lan2" "switch0" "0x04" "" "link"
+	ucidef_set_led_switch "lan3" "lan3" "amber:lan3" "switch0" "0x08" "" "link"
+	ucidef_set_led_switch "lan4" "lan4" "amber:lan4" "switch0" "0x10" "" "link"
+	ucidef_set_led_switch "wan" "wan" "amber:wan" "switch0" "0x01" "" "link"
+	ucidef_set_led_netdev "internet" "internet" "green:internet" "eth0.2"
```

Comment:
WAN and Internet seem redundant, and the switch-based trigger is preferred. Probably it's best to simply drop the internet LED here.

Reply:
Okay, I'll delete it.